remix/Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: up start_db
+
+up:
+	make start_db
+	npm run dev
+
+down:
+	make stop_db
+	
+start_db:
+	docker-compose up -d
+	
+stop_db:
+	docker-compose down -v
```

Comment:
Containers should be stopped only with `docker-compose stop`; `docker-compose down` would remove the container entirely.